components/menu/README.md

```diff
@@ -0,0 +1,146 @@
+# Menu
+
+## d2l-menu
+
+A basic menu can be defined using `d2l-menu` and a combination of menu items (e.g., `d2l-menu-item`, `d2l-menu-item-separator`).  **Important**: specify a label on your `d2l-menu` for screen-readers.
+
+![Menu](./screenshots/menu.png?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/menu/menu.js';
+</script>
+
+<d2l-menu label="Astronomy">
+  <d2l-menu-item text="Introduction"></d2l-menu-item>
+  <d2l-menu-item-separator></d2l-menu-item-separator>
+  <d2l-menu-item text="Searching the Heavens"></d2l-menu-item>
+  <d2l-menu-item text="The Solar System"></d2l-menu-item>
+  <d2l-menu-item text="Stars &amp; Galaxies"></d2l-menu-item>
+  <d2l-menu-item text="The Night Sky"></d2l-menu-item>
+  <d2l-menu-item text="The Universe"></d2l-menu-item>
+</d2l-menu>
+```
+
+**Properties:**
+
+* `label` (required, String): Text to be applied to menu `aria-label` for use with screen readers
+
+**Events:**
+
+* `d2l-menu-resize`: dispatched when size of menu changes (e.g., when nested menu of a different size is opened)
+
+**Note:** `d2l-menu` renders without an outer border since it's typically used in a context where a containing element defines a border (ex. `d2l-dropdown-menu` or side nav).
+
+## Menu Items
```

Comment:
I might be tempted to reorganize this part of the readme, so that each menu item type has it's own section.
```
## d2l-menu-item
...

## d2l-menu-item-link
...

## d2l-menu-item-radio
...

## d2l-menu-item-checkbox
...
```
🤷‍♂